python_modules/libraries/dagster-snowflake-pandas/dagster_snowflake_pandas/snowflake_pandas_type_handler.py

```diff
@@ -111,8 +85,16 @@ def handle_output(
 
     def load_input(self, context: InputContext, table_slice: TableSlice) -> pd.DataFrame:
         with _connect_snowflake(context, table_slice) as con:
-            result = pd.read_sql(sql=SnowflakeDbClient.get_select_statement(table_slice), con=con)
-            result = result.apply(_convert_string_to_timestamp, axis="index")
+            try:
+                result = pd.read_sql(sql=SnowflakeDbClient.get_select_statement(table_slice), con=con)
+            except InterfaceError as e:
```

Comment:
if we go down the route of https://github.com/dagster-io/dagster/pull/12190/files#r1100731596 this error check would get removed